docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -19,54 +19,42 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 
 When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/deployment/add-semgrep-to-ci).
 
-To split up your monorepo, you need to make two changes. First, use the `--include` flag to determine *how* you want to logically split up the code. Second, update the `SEMGREP_REPO_DISPLAY_NAME` environment variable to assign findings to separate projects in Semgrep AppSec Platform. 
+There are two tools provided to split up a repo. Consider a monorepo named `monorepo` with four main modules
 
-For example, if the monorepo has four main modules and their paths are:
-```
-src/moduleA
-src/moduleB
-src/moduleC
-src/moduleD
-```
-
-Then splitting its scans into four separate scans, one for each module, would provide a logical separation for findings. In general, we recommend that modules not exceed ~100,000 lines of code in order to maintain optimal scan time and efficiency.
-
-After choosing a logical split, use the `--include` flag ([see CLI reference](/docs/cli-reference)) with the relevant path to only scan files in that module's code path:
+    /src/moduleA
+    /src/moduleB
+    /src/moduleC
+    /src/moduleD
 
-```
-semgrep ci --include=src/moduleA/**
-```
+The easiest way to split this monorepo up is into four separate scans, one for each module. To do this, use the `--subdir` (see `semgrep ci --help`) flag with the relevant path to only scan files in that module's code path:
 
-Now, Semgrep is only scanning files under that path and the CI run will take less time, since less code is being scanned.
+    semgrep ci --subdir /src/moduleA/*
 
-For the other modules, the commands look similar. For module B:
+In addition to scanning `/src/moduleA/*`, this will send the results to a project called `monorepo/src/moduleA`. If you want to change the project name, just set the `SEMGREP_REPO_DISPLAY_NAME` environment variable, available since Semgrep version 1.61.1.
```

Comment:
```suggestion
In addition to scanning `/src/moduleA/*`, this command sends the results to a project called `monorepo/src/moduleA`. If you want to change the project name, set the `SEMGREP_REPO_DISPLAY_NAME` environment variable, available since Semgrep version 1.61.1.
```